ux-microservice/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
If using JS-not-TS, then we don't need this file

Reply:
yes but we don't know yet until tomorrow

Reply:
I was hoping to go TypeScript with this too so just got it ready.